src/GafferAppleseed/IECoreAppleseedPreview/Renderer.cpp

```diff
@@ -1576,6 +1577,12 @@ class AppleseedRenderer : public IECoreScenePreview::Renderer
 				return;
 			}
 
+			if( name == g_frameOptionName )
+			{
+				/// \todo Does this have a meaning in Appleseed?
```

Comment:
We don't have a setting for the current frame, but I was thinking now that we can add one and then I can fill the details here in a future PR.